### ISSUE TITLE
Some python files in pylint/ project don't have proper license header

### ISSUE THREAD

**Issue by nedn** [PyCQA/pylint#894]
Hi,
I am trying to add pylint/ project to Chromium project (https://codereview.chromium.org/1920403002/). However, the licensechecker of Chromium complains about some python files in pylint/ don't have proper license header, those are:
`pylint/pylint/__main__.py`
`pylint/pylint/pyreverse/__init__.py` 
`pylint/pylint/reporters/guireporter.py` 

Can someone update these files with license header? Thanks a lot!

**Comment by PCManticore** [comment]
Sure, thanks for letting us know!

**Comment by PCManticore** [comment]
Should be done now. Let me know if you still have issues with it.

**Comment by sandrotosi** [comment]
sorry but this change is wrong! you removed the copyright information, which is a vital one for legal reason: a license is the way you grant usage rights to the given source code, the copyright is who has the right to assign that license - now the latter part is missing in the source files and it's not reported anywhere clearly. grepping one can find:

```
pylint/checkers/misc.py:""" Copyright (c) 2000-2010 LOGILAB S.A. (Paris, FRANCE).
pylint/checkers/raw_metrics.py:""" Copyright (c) 2003-2010 LOGILAB S.A. (Paris, FRANCE).
pylint/config.py:        if hasattr(pkginfo, "copyright"):
pylint/config.py:.SH COPYRIGHT
pylint/config.py:''' % pkginfo.copyright
pylint/test/functional/wrong_import_position15.py:__copyright__ = 'Some copyright'
setup.py:# copyright 2003-2013 LOGILAB S.A. (Paris, FRANCE), all rights reserved.
```

(and setup.py still has the old license test) which is contrast with the previously reported copyright notice (back in 1.5.6):

```
Copyright: Copyright (c) 2003-2010 Sylvain Thenault (thenault@gmail.com).
Copyright (c) 2000-2015 LOGILAB S.A. (Paris, FRANCE).
```

can someone please look into this asap? it's preventing to upgrade pylint in debian

**Comment by PCManticore** [comment]
From what you are saying, we just need to put the copyright somewhere, not necessarily in each file, no? Can it be placed into the LICENSE file, would this solve your problem? Would it need a release for upgrading into Debian?

**Comment by sandrotosi** [comment]
it could be in just one file, f.e. `COPYING` in the toplevel dir of the tarball, but it has to be released among the other source code files, so that code/license/copyright are all released together.

to recap: 1. yes in one file is fine, 2. yes we need a new release

sorry about that :(

**Comment by PCManticore** [comment]
Okay, I'll do it now, thanks for letting me know about this. Shouldn't take too much.

**Comment by sandrotosi** [comment]
thanks a ton!!

**Comment by PCManticore** [comment]
For the record, I'm attempting to fix this in https://github.com/PyCQA/pylint/commit/587776beeee88a2e3e5b143b1dc6cdcba49e5236. Waiting for travis before releasing, although let me know if it looks okay before doing the actual release.

**Comment by sandrotosi** [comment]
i commented on that commit, but it looks good - thanks for addressing this so quickly!

**Comment by PCManticore** [comment]
@sandrotosi 1.6.4 should be out now. Let me know if this fixes the problem.

**Comment by sandrotosi** [comment]
thanks @PCManticore !! All looks good now! (well, i still cant package 1.6.4 but due to missing deps :) )

**Comment by PCManticore** [comment]
No worries, thanks for bringing it up. Are those deps managed by us (e.g.
astroid) or are they third party libraries, which we use? Let me know if
there is something on our side that needs to be addressed.

**Comment by sandrotosi** [comment]
for now i stubled upon isort, which Debian packages only for py3k, but we need the module for python2 as well, so nothing on your side, and thanks for checking!
